hpx/parallel/executors/execution_fwd.hpp

```diff
@@ -67,6 +68,7 @@ namespace hpx { namespace parallel { namespace execution
             }
         };
 
+#ifdef HPX_HAVE_CXX14_AUTO
```

Comment:
Please don't use a macro named `_CXX14_...` to guard code which is C++11 compliant.